packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -214,7 +215,10 @@ const errorsContent = (
 
 const overviewContent = (
   <Grid container spacing={3} alignItems="stretch">
-    <Grid item md={6}>
+    <Grid item xs={12}>
+      <EntityOrphanWarning />
```

Comment:
But awkward to have to add this manually, but also pretty nice and explicit and doesn't take too much space. I guess we can see how this list grows. It does also set a precedent for how to handle these kind of additions, and I think this would be a good model to use and is aligned what we've seen the need for internally as well.

Reply:
Yeah, adding manually isn't that nice. Maybe this could also be part of the entity page wrapper itself? But that would also force a different location...

Reply:
Yeah it becomes a bit tricky to configure then, but more importantly the entity page has no idea what an "overview" page is, so we'd have to add it in the header or on every page or something like that